core/cmd/admin_commands.go

```diff
@@ -195,6 +196,11 @@ func (s *Shell) ListUsers(_ *cli.Context) (err error) {
 
 // CreateUser creates a new user by prompting for email, password, and role
 func (s *Shell) CreateUser(c *cli.Context) (err error) {
+	// Check user's email validity. Note that it will also be later checked on the server side in the NewUser function.
+	if err := sessions.ValidateEmail(c.String("email")); err != nil {
```

Comment:
`err` is already declared in the return parameters
```suggestion
	if err = sessions.ValidateEmail(c.String("email")); err != nil {
```